query-engine/connector-test-kit-rs/query-engine-tests/tests/query_engine_tests.rs

```diff
@@ -1,6 +1,6 @@
 #![allow(clippy::module_inception, clippy::too_many_arguments)]
 
 mod new;
-mod queries;
-mod raw;
-mod writes;
+// mod queries;
+// mod raw;
+// mod writes;
```

Comment:
Would you please remove these rather than comment them out? They are left in the git history in case they are needed.